src/main/java/org/folio/circulation/infrastructure/storage/AutomatedPatronBlocksRepository.java

```diff
@@ -22,7 +22,7 @@ public AutomatedPatronBlocksRepository(Clients clients) {
 
   public CompletableFuture<Result<AutomatedPatronBlocks>> findByUserId(String userId) {
     if(isNull(userId)) {
-      return ofAsync(() -> null);
+      return ofAsync(AutomatedPatronBlocks::new);
```

Comment:
Should this be handled on the next step?

Reply:
Done